packages/lesswrong/lib/collections/users/newSchema.ts

```diff
@@ -1432,6 +1432,25 @@ const schema = {
       },
     },
   },
+  // Users who are blocked from messaging this user
+  blockedUserIds: {
+    database: {
+      type: "VARCHAR(27)[]",
+      defaultValue: [],
+      canAutofillDefault: true,
+      nullable: false,
+    },
+    graphql: {
+      outputType: "[String!]",
+      inputType: "[String!]",
+      canRead: [userOwns, "sunshineRegiment", "admins"],
+      canUpdate: [userOwnsAndInGroup("trustLevel1"), "sunshineRegiment", "admins"],
```

Comment:
```suggestion
      canUpdate: [userOwns, "sunshineRegiment", "admins"],
```
I don't think we should require "trustLevel1" for this. I don't know if we consistently apply that group still, and unlike the personal blog case there is no public facing change resulting from this so I think anyone should be able to do it